src/com/epam/javatrainings/classdecomposition/palmetto/Order.java

```diff
@@ -0,0 +1,58 @@
+package com.epam.javatrainings.classdecomposition.palmetto;
+
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Random;
+
+public class Order {
+    private List<Pizza> pizza;
+    private static int orderNumber;
+    private static final List<Integer> collectionOfOrderNumber;
+    private final LocalTime localTime;
+
+    static {
+        collectionOfOrderNumber = new LinkedList<>();
+    }
+
+    {
+        this.localTime = LocalTime.now();
```

Comment:
why not in constructor?

Reply:
Just the Idea did not feel good about it had been in the constructor.